style.css

```diff
@@ -0,0 +1,608 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  border: 0;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+}
+
+img {
+  cursor: pointer;
+}
+
+.hamburger-button {
+  margin-bottom: 5%;
+}
+
+.mobilemenu {
+  display: none;
+}
+
+.head-padding {
+  padding: 4% 5%;
+}
+
+.noone1 {
+  display: none;
+}
+
+.about-section {
+  background-color: #d3d3d3;
+  padding: 3%;
+  border: 3px solid white;
+  font-size: 18px;
+  margin-bottom: 25px;
+}
+
+.abt-p {
+  background-color: white;
+}
+
+.first-card {
+  background-image: url(images/Background-image.jpg);
+  background-size: contain;
+}
+
+.first-card h2 {
+  color: #ec5242;
+  font-size: 24px;
+  font-weight: 500;
+  margin-bottom: 10px;
+}
+
+.first-card h1 {
+  font-size: 46px;
+  color: #ec5242;
+  font-weight: 800;
+  margin-bottom: 50px;
+  background: linear-gradient(rgb(212, 41, 41), #ec5242);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.first-card .date-section {
+  color: #272a31;
+  font-size: 29px;
+  font-weight: 900;
+}
+
+.venue {
+  color: #272a31;
+  font-size: 18px;
+  padding-bottom: 10%;
+}
+
+.main-section {
+  background-color: #272a31;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.main-section ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] [OPTIONAL] a lot of the code in your CSS file is repeated for example the display flex is repeated as you can see in this example, you could make your code a lot shorter and more readable if you made 1 class for flex-box for example.